app/src/main/java/pivx/org/pivxwallet/ui/base/BaseRecyclerFragment.java

```diff
@@ -75,6 +77,21 @@ public void onRefresh() {
                     }
                 }
         );
+
+        imgEmptyView.setOnClickListener(new View.OnClickListener() {
```

Comment:
This class is called BaseRecyclerFragment, don't add specific logic here. Move your logic down in the class hierarchy.

Reply:
We have removed logic "imgEmptyView.setOnClickListener" from BaseRecyclerFragment now.

https://github.com/PIVX-Project/PIVX-Android/pull/8/commits/359d68e08713226fd43ee5d46ade5cf55b767ca7